CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+### citus v9.3.1 (Jun 17, 2020) ###
+
+* Adds support to partially push down tdigest aggregates
+
+* Fixes a crash that occurs inserting implicitly coerced constants
+
+### citus v9.2.5 (Jun 17, 2020) ###
+
+* Adds support to partially push down tdigest aggregates
+
+* Fixes a bug with generated columns not at the end of the column list
```

Comment:
We had an item for this on `v9.3.0`. I think we should have the same item here for consistency.

> Fixes an issue with distributing tables having generated cols not at the end